iron-a11y-keys-behavior.html

```diff
@@ -249,7 +249,9 @@
     Polymer.IronA11yKeysBehavior = {
       properties: {
         /**
-         * The HTMLElement that will be firing relevant KeyboardEvents.
+         * The HTMLElement that will be firing relevant KeyboardEvents. Set it to
+         * `null` to disable the listeners.
+         * @type {EventTarget}
```

Comment:
nit: `@type {?EventTarget}` to make it clearer it can (and will) be `null` in some cases